src/components/AuthPanel/AuthPanel.tsx

```diff
@@ -22,6 +22,7 @@ const AuthPanel: FC = () => {
   const dispatch = useAppDispatch();
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const openUserMenu = Boolean(anchorEl);
+  const isOpenDialog = useAppSelector((store) => store.dialogSlice.isOpen);
```

Comment:
You can also read about selectors in toolkit. It allows you to reduce the amount of code in your component.

Reply:
Thanks. I will try.